build/webpack.pro.config.js

```diff
@@ -4,6 +4,11 @@ const webpack = require('webpack');
 const config = {
   mode: 'production',
   devtool: false,
+  performance: {
+    hints: false,
+    maxEntrypointSize: 2500000,
+    maxAssetSize: 2500000
```

Comment:
Looks like `maxEntrypointSize` and `maxAssetSize` is useless since hints is disabled?

Reply:
Yes, indeed.
`hints: false,` line should be deleted